gef.py

```diff
@@ -77,28 +77,17 @@
 import tempfile
 import time
 import traceback
-import configparser
-import xmlrpc.client as xmlrpclib
 import warnings
-
+import xmlrpc.client as xmlrpclib
 from functools import lru_cache
 from io import StringIO
+from types import ModuleType
+from typing import (List, Dict, Tuple, Optional, Sequence, Union, Generator, Any, Iterator,
+                    Callable, ByteString, Set, Type)
 from urllib.request import urlopen
```

Comment:
```suggestion
from typing import (List, Dict, Tuple, Optional, Sequence, Union, Generator, Any, Iterator,
                    Callable, ByteString, Set, Type, NoReturn)
```

This suggested change is to correctly modify the signature of `FakeExit`as suggested further down in this file.